css/null.css

```diff
@@ -0,0 +1,32 @@
+*,*::before,*::after{

```

Comment:
i wonder why this file is called `null`. Does is mean a basic set of styles for main html tags?

Reply:
> i wonder why this file is called `null`. Does is mean a basic set of styles for main html tags?

this file is called 'null', because it makes nullification of basic set of styles in each browser. I've downloaded it from the Internet, there are many properties i don't know about, but it is really useful